rclcpp_action/test/test_client.cpp

```diff
@@ -272,7 +272,8 @@ TEST_F(TestClient, construction_and_destruction)
 
 TEST_F(TestClient, async_send_goal_no_callbacks)
 {
-  auto action_client = rclcpp_action::create_client<ActionType>(client_node, action_name);
+  auto action_client = rclcpp_action::create_client<ActionType, rclcpp::Node>(client_node,
```

Comment:
I may be wrong, but I don't think we need to specify the template argument:
```suggestion
  auto action_client = rclcpp_action::create_client<ActionType>(client_node,
```

Reply:
This is to explicitly provide a working test of the template, you're 100% right  its not necessary

Reply:
If it's test the construction with a provided template argument, then I suggest adding an extra test to the `construction_and_destruction` test above (L270). It's seems like a more relevant place to test this.

Reply:
Fair enough. I actually did do that but theres a bug in gtest that doesn't allow multiple template arguments in an assert. Its a known problem google filed and then closed a ticket with `won't fix`

Reply:
It's okay if  you don't wrap it in an assert macro. Simply calling the method should be enough (if it throws I think we'll notice).

Reply:
10-4